src/store/index.js

```diff
@@ -1,33 +0,0 @@
-import {
```

Comment:
is there a reason we're moving store?  I thought we only wanted to reuse certain components, like the Table

Reply:
Yeah, because the other web app also uses the same store code. I am basically moving all the common stuff, not only components.

Reply:
The store structures probably won't match though, since there are definitely dota-specific things in there like abilities and heroes?

Reply:
Yup, store structures won't match, which is why there still is `import app from '../reducers';`

Reply:
Could we keep separate store files? I think it's possible that we might want to make changes to one or the other.

Reply:
I think we should try to share only components and not framework code

Reply:
We can always put the separate store file when we need it. Until then we can use it as a common part. When you want to share code between 2 websites, it doesn't have to be components only.